docs/luigi-compound-container-api.md

```diff
@@ -31,7 +31,7 @@ Type: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Globa
 ### compoundConfig
 
 The configuration for the compound microfrontend
```

Comment:
.
```suggestion
The configuration for the compound microfrontend.
```

Just saw that a dot seem to have been missing here. You can just commit the change.